index.bs

```diff
@@ -231,57 +231,57 @@ Given a task |task|, perform the following algorithm:
 
 2. Let |destinationRealms| be an empty set.
 
-3. Determine the set of JavaScript Realms to which reports will be delivered:
+3. Determine the set of <a href="https://tc39.github.io/ecma262/#sec-code-realms">JavaScript Realms</a> to which reports will be delivered:
 
-    For each environment settings object |settings| in task's script evaluation environment settings object set:
+    For each <a>environment settings object</a> |settings| in |task|'s [=task/script evaluation environment settings=] object set:
```

Comment:
Looks like a preexisting problem, but "object set" should still be part of the definition/link

Reply:
Done